test/e2e-olm/kibana_test.go

```diff
@@ -0,0 +1,118 @@
+package e2e
+
+import (
+	goctx "context"
+	"fmt"
+	"testing"
+	"time"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	"github.com/openshift/elasticsearch-operator/pkg/k8shandler/kibana"
+	"github.com/openshift/elasticsearch-operator/test/utils"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestKibana(t *testing.T) {
+	registerSchemes(t)
+	t.Run("kibana-group", func(t *testing.T) {
+		t.Run("KibanaDeployment", KibanaDeployment)
+	})
+}
+
+func KibanaDeployment(t *testing.T) {
+	ctx := framework.NewTestCtx(t)
+
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Logf("Found namespace: %q", namespace)
+
+	// get global framework variables
+	f := framework.Global
+	// wait for elasticsearch-operator to be ready
+	err = e2eutil.WaitForDeployment(t, f.KubeClient, namespace, "elasticsearch-operator", 1, retryInterval, timeout)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if err = createElasticsearchSecret(f, ctx); err != nil {
+		t.Fatal(err)
+	}
+
+	uuid := utils.GenerateUUID()
+	esDeploymentName := fmt.Sprintf("elasticsearch-cdm-%v-1", uuid)
+	_, err = createElasticsearchCR(f, ctx, uuid)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = e2eutil.WaitForDeployment(t, f.KubeClient, namespace, esDeploymentName, 1, retryInterval, timeout)
+	if err != nil {
+		t.Errorf("timed out waiting for Deployment %q: %v", esDeploymentName, err)
+	}
+
+	if err = createTrustedCABundle(f, ctx); err != nil {
```

Comment:
we don't need to create the trusted CA bundle -- the operator and k8s should be handling this for us.

Reply:
Alright, i drop it.